src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return function getSum(number1) {
+    countCalls++;
+
+    return (number2) => {
```

Comment:
Keep consistent code style.
```suggestion
  return (number1) => {
    countCalls++;

    return (number2) => {
```